swap_meet/vendor.py

```diff
@@ -1,2 +1,64 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        try:
+            self.inventory.remove(item)
+            return item
+        except ValueError:
+            return False
+
+    def get_by_category(self, category):
+        """gets items in inventory of given category"""
+        items_by_category = []
+        
+        for item in self.inventory:
+            if item.category == category:
+                items_by_category.append(item)
+
+        return items_by_category
+    
+    def swap_items(self, other, my_item, their_item):
+        if my_item not in self.inventory or their_item not in other.inventory:
+            return False
+
+        my_index = self.inventory.index(my_item)
+        their_index = other.inventory.index(their_item)
+
+        self.inventory[my_index], other.inventory[their_index] = other.inventory[their_index], self.inventory[my_index]
```

Comment:
👍  Nice tuple swap.

Writing
```python
    a, b = c, d
```

is essentially equivalent to 
```python
    new_tuple = c, d
    a = new_tuple[0]
    b = new_tuple[1]
```

but can be a little more efficient.